code/src/App.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
+import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom';
+// import Header from 'components/Header';//
+import Library from 'components/Library';
+import Details from 'components/Details';
+import NotFound from 'components/NotFound';
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Library library={Library} />} />
+        <Route path="/details/:id" element={<Details />} />
+        <Route path="/404" element={<NotFound />} />
```

Comment:
Nice with a 404-page! We couldn't get it to work, but neither of the ones we tried worked, so it is probably not on your end (: